internal/importers/base/importer.go

```diff
@@ -291,3 +299,20 @@ func (i *Importer) convertToConstraint(v gps.Version) gps.Constraint {
 	}
 	return v
 }
+
+func (i *Importer) isDefaultSource(projectRoot gps.ProjectRoot, sourceURL string) (bool, error) {
+	if sourceURL == "https://"+string(projectRoot) {
```

Comment:
Can you help me understand why this is necessary? I would have thought that `SourceURLsForPath` would cover this as well.

Reply:
mainly for `gopkg.in` imports, as for example importing `gopkg.in/yaml.v2` via `gvt` will appear in the manifest as:
```json
{
	"importpath": "gopkg.in/yaml.v2",
	"repository": "https://gopkg.in/yaml.v2",
	"vcs": "",
	"revision": "f7716cbe52baa25d2e9b0d0da546fcf909fc16b4",
	"branch": "v2"
}
```
but `SourceURLsForPath()` will return `[https://github.com/airbrake/gobrake http://github.com/airbrake/gobrake]`

In addition, this small condition will usually return true and will same save some CPU cycles :smile:

Reply:
Would you please add a comment explaining that we are essentially checking for gopkg.in URLs?